src/main/webapp/assets/modules/common/autoSuggestion.js

```diff
@@ -4,12 +4,20 @@ techlooper.directive('autoSuggestion', function ($http) {
     scope: {
       text: "=",
       items: "=",
+      inputModel: "=",
       getUrl: "@"
     },
     link: function (scope, element, attr, ctrl) {
-      scope.$watch("text", function (newVal) {
-        if (!newVal) return;
+      scope.$watch("inputModel", function (newVal, oldVal) {
+        //console.log(scope.inputModel);
+      });
+
+      scope.$watch("text", function (newVal, oldVal) {
```

Comment:
Mixed double and single quotes.<br>'oldVal' is defined but never used.<br>'newVal' is defined but never used.